### DISCUSSION TITLE
Turbo with Javascript instead of TypeScript

### DISCUSSION THREAD

**Discussion by ahmad5149** [vercel/turborepo#8071] [Help] [answered]
### Goals

I want to create the **turbo** with javascript not typescript. 

### Non-goals

1.
2.
3.

### Background

We are using the 3 Next Js apps, which are in JS, not the typescript. How can I convert these into the turbo. 

### Proposal

Yes, I am interested.

**Comment by versecafe** [comment, marked as answer]
@ahmad5149 turbo repo has no requirements that you use Typescript, you can set up a normal turbo repo, and NextJS still has a build step either way that you set up the same way, all NextJS typescript compilation happens during "next build" so you should be able to just drop in a JS NextJS project the same way you would a TS one
